torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -1557,6 +1557,17 @@ void InitXlaModuleBindings(py::module m) {
           result_tuple[1] = new_token;
           return result_tuple;
         });
+  m.def(
+      "_xla_spmd_reduce_scatter",
```

Comment:
so the only difference is this one does not have token?

Reply:
That's one difference. Others are mentioned in the description.